webpack/scenes/Hosts/ChangeContentSource/helpers.js

```diff
@@ -15,6 +15,11 @@ export const getHostIds = () => {
   return [];
 };
 
+export const getSearchParam = () => {
+  const url = new URL(window.location);
+  return (url.searchParams.get('search'));
+};
```

Comment:
You may consider reusing `useUrlParams` from [here](https://github.com/Katello/katello/blob/6b36dea1b9be7208c89a468703f7f1d9c56045c5/webpack/components/Table/TableHooks.js#L259); it has some URL decoding and other niceties that you might find useful :)